index.js

```diff
@@ -27,9 +27,7 @@ function GoogleAssistant(authConfig, callback) {
   if (authConfig.oauth2Client) {
     // we are passing in a client that is already authed with Google
     assistant = new Assistant(authConfig.oauth2Client);
-    process.nextTick( () => {
-      assistantReady();
-    });
+    process.nextTick(assistantReady());
```

Comment:
change: `process.nextTick(assistantReady());`
to: `process.nextTick(assistantReady);`

nextTick takes a function as the parameter, not the result of the function

Reply:
Sorry, my bad. Was just a typo because lack of time! Thanks for pointing it out and seeing it. 
I will fix it in the evening.